# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcProxyChannelParent.h
# Commit: c684f79d8c67
# Full Hash: c684f79d8c673f21179fce00358d70845a9692c0
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Rename PWebrtcProxyChannel to PWebrtcTCPSocket. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45098
# ==============================================================================

diff -r 82af7b115f60 -r c684f79d8c67 media/mtransport/ipc/WebrtcProxyChannelParent.h
--- a/media/mtransport/ipc/WebrtcProxyChannelParent.h	Tue Sep 17 18:15:23 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,60 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set sw=2 ts=8 et tw=80 ft=cpp : */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef mozilla_net_WebrtcProxyChannelParent_h
-#define mozilla_net_WebrtcProxyChannelParent_h
-
-#include "mozilla/net/PWebrtcProxyChannelParent.h"
-
-#include "WebrtcProxyChannelCallback.h"
-
-class nsIAuthPromptProvider;
-
-namespace mozilla {
-namespace net {
-
-class WebrtcProxyChannel;
-
-class WebrtcProxyChannelParent : public PWebrtcProxyChannelParent,
-                                 public WebrtcProxyChannelCallback {
- public:
-  NS_INLINE_DECL_THREADSAFE_REFCOUNTING(WebrtcProxyChannelParent, override)
-
-  mozilla::ipc::IPCResult RecvAsyncOpen(const nsCString& aHost,
-                                        const int& aPort,
-                                        const LoadInfoArgs& aLoadInfoArgs,
-                                        const nsCString& aAlpn) override;
-
-  mozilla::ipc::IPCResult RecvWrite(nsTArray<uint8_t>&& aWriteData) override;
-
-  mozilla::ipc::IPCResult RecvClose() override;
-
-  void ActorDestroy(ActorDestroyReason aWhy) override;
-
-  explicit WebrtcProxyChannelParent(dom::TabId aTabId);
-
-  // WebrtcProxyChannelCallback
-  void OnClose(nsresult aReason) override;
-  void OnConnected() override;
-  void OnRead(nsTArray<uint8_t>&& bytes) override;
-
-  void AddIPDLReference() { AddRef(); }
-  void ReleaseIPDLReference() { Release(); }
-
- protected:
-  virtual ~WebrtcProxyChannelParent();
-
- private:
-  void CleanupChannel();
-
-  // Indicates that IPC is open.
-  RefPtr<WebrtcProxyChannel> mChannel;
-};
-
-}  // namespace net
-}  // namespace mozilla
-
-#endif  // mozilla_net_WebrtcProxyChannelParent_h